content/en/docs/access-control-and-account-management/external-authentication/use-an-ldap-service.md

```diff
@@ -12,8 +12,21 @@ This document describes how to use an LDAP service as an external identity provi
 ## Prerequisites
 
 * You need to deploy a Kubernetes cluster and install KubeSphere in the cluster. For details, see [Installing on Linux](/docs/installing-on-linux/) and [Installing on Kubernetes](/docs/installing-on-kubernetes/).
+
 * You need to obtain the manager distinguished name (DN) and manager password of an LDAP service.
 
+* If you are using KubeSphere 3.2.0, run the following command before configuring LDAP:
```

Comment:
```suggestion
* If you are using KubeSphere 3.2.0, run the following command after configuring LDAP and waiting until ks-install run successfully:
```

Reply:
Updated in [this commit](https://github.com/kubesphere/website/pull/2114/commits/0066310ba9572a4d23638705e4b22e4a5d26ec5a).